website/docs/r/org.html.markdown

```diff
@@ -30,6 +30,17 @@ resource "vcd_org" "my-org" {
   is_enabled       = "true"
   delete_recursive = "true"
   delete_force     = "true"
+
+  vapp_lease {
+    maximum_runtime_lease_in_sec          = 3600  // 1 hour
```

Comment:
Syntax highlighting is off when using "//" for comments, but maybe it's only when running locally? If not, "#" seems to work fine.

![Screenshot 2020-01-17 at 16 49 24](https://user-images.githubusercontent.com/11330689/72621175-6cdc6380-3949-11ea-80d3-7f893400e7e7.png)

Reply:
Changed